### PR TITLE
removed warnings

### PR DESCRIPTION
**Pull request by mauro-fanton**

### CODE REVIEW

File: app/connectors/addresslookup/AddressLookupFrontendConnector.scala [hunk 1]
```diff
@@ -49,8 +49,9 @@ class AddressLookupFrontendConnector @Inject() (
         case response @ HttpResponse(ACCEPTED, _, _) =>
           response.header(LOCATION) match {
             case Some(redirectUrl) => AddressLookupOnRamp(redirectUrl)
-            case None              => throw new IllegalStateException("Missing re-direct url")
+            case _              => throw new IllegalStateException("Missing re-direct url")
           }
+        case error => throw new IllegalStateException(s"Error response with status: ${error.status}")
```
[the remark this answers is not in the document]
Comment (lmcewanhmrc) [reply app/connectors/addresslookup/AddressLookupFrontendConnector.scala:54]: Would be useful to include where the error came from, ie whichever bit of ALF / the API we're calling

**Review by liampan — APPROVED** [review]
Has def getRequest() been deleted? I cant see where it was coming from but it looks to be no longer used

File: app/connectors/addresslookup/AddressLookupFrontendConnector.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (mauro-fanton) [reply app/connectors/addresslookup/AddressLookupFrontendConnector.scala]: done. Thanks

File: app/controllers/partner/PartnerTypeControllerBase.scala [hunk 2]
```diff
@@ -16,37 +16,19 @@
...
-  GENERAL_PARTNERSHIP,
-  LIMITED_LIABILITY_PARTNERSHIP,
-  OVERSEAS_COMPANY_UK_BRANCH,
-  REGISTERED_SOCIETY,
-  SCOTTISH_LIMITED_PARTNERSHIP,
-  SCOTTISH_PARTNERSHIP,
-  SOLE_TRADER,
-  UK_COMPANY
-}
+import forms.organisation.PartnerTypeEnum.{GENERAL_PARTNERSHIP, LIMITED_LIABILITY_PARTNERSHIP, OVERSEAS_COMPANY_UK_BRANCH, REGISTERED_SOCIETY, SCOTTISH_LIMITED_PARTNERSHIP, SCOTTISH_PARTNERSHIP, SOLE_TRADER, UK_COMPANY}
```
[the remark this answers is not in the document]
Comment (mauro-fanton) [reply app/controllers/partner/PartnerTypeControllerBase.scala]: yep, agree. done. Thanks

**Review by liampan — APPROVED** [review]

**Pull request merged**